tensorflow_datasets/image/oxford_iiit_pet.py

```diff
@@ -102,7 +102,6 @@ def _split_generators(self, dl_manager):
     # Setup train and test splits
     train_split = tfds.core.SplitGenerator(
         name="train",
-        num_shards=_NUM_SHARDS,
```

Comment:
Note that you could also remove `_NUM_SHARDS = 123` at the top of the file, which isn't used anymore.

Reply:
Yeah, I noticed that.